configurations/netvm/default.nix

```diff
@@ -0,0 +1,82 @@
+{
+  nixpkgs,
+  microvm,
+  system,
+}:
+nixpkgs.lib.nixosSystem {
+  inherit system;
+  modules = [
+    # TODO: Enable only for development builds
+    ../../modules/development/authentication.nix
+    ../../modules/development/ssh.nix
+    ../../modules/development/packages.nix
+
+    microvm.nixosModules.microvm
+
+    ({pkgs, ...}: {
+      networking.hostName = "netvm";
+      system.stateVersion = "22.11";
```

Comment:
should we inherit the `stateVersion` from the host as well? it's ok to mark that with `# TODO` as well

Reply:
I'll mark it with # TODO.

The stateVersion largely does not affect us anyway, since at least currently we are not keeping any state. At least in the current development stage we could just live with the default. I'm just setting this to get rid of the warning. The stateVersion is only a check for some individual packages, if the structure of the state under /var/lib has changed. For example if you update your NixOS from some old version like 19 all the way to 23.05, then some services might expect to find their files from different path

Reply:
Also the guest could have different stateVersion and technically we could use different nixpkgs-versions for individual microvms